Form/Type/ColorSelectorType.php

```diff
@@ -48,7 +48,15 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::WHITE,
                 Colors::YELLOW,
             ),
-        ));
+        );
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $defaults['choices'] = array_unique(array_flip(Colors::getAll()));
```

Comment:
I think I will say a big bullshit right now. Coding without really understand what should be done. It would be something like this?
```php
    /**
     * Return the list of colors.
     *
     * @return array
     */
    public static function getAll($isInverted = true)
    {
        if (func_num_args() == 0) {
            @trigger_error(
                'Omit $isInverted argument of '.__METHOD__.' is deprecated since 3.3, to be removed in 4.0.',
                E_USER_DEPRECATED
            );
        }
        if($isInverted) {
                return array(
                        'aliceblue' => self::ALICEBLUE,
                        'antiquewhite' => self::ANTIQUEWHITE,
                        //....
                );
        }
        return array(
            self::ALICEBLUE => 'aliceblue',
            self::ANTIQUEWHITE => 'antiquewhite',
            //....
        );
    }
```